lua/devcontainer_cli/config/init.lua

```diff
@@ -18,6 +18,11 @@ local default_config = {
   -- Remove existing container each time DevcontainerUp is executed
   -- If set to True [default_value] it can take extra time as you force to start from scratch
   remove_existing_container = true,
+  -- nvim_dotfiles that will be installed inside the docker devcontainer throught the devcontainer cli.
+  -- [default_value] - LazyVim/starter
+  nvim_dotfiles = "https://github.com/LazyVim/starter",
```

Comment:
by default LazyVim/starter config is being used:
https://github.com/LazyVim/starter